api/marketplace.py

```diff
@@ -19,6 +19,10 @@ def get_offers(self, include_empty_offers: bool = False) -> List[Offer]:
         r = self.request('get', 'offers', params={'include_empty_offer': include_empty_offers})
         return Offer.from_list(r.json())
 
+    def get_update_rates(self):
+        r = self.request('get', 'config')
+        return {key: value for (key, value) in r.json().items() if 'max' in key}
```

Comment:
This could be more explicit. I prefer returning `max_updates_per_sale` and `max_req_per_sec` as tuple (or even as a dedicated object). Alternatively, there could be a function for each of the values.

(The merchant should not get infos about `consumer_per_minute`. We should remove this from this API call. But that's for another PR)

Reply:
As far as I have looked through the MP code, I don't see the `consumer_per_minute` used anyhow. So removing it from the API would definitely make sense, maybe even removing it from the marketplace?

The tuple does not make too much sense to me. I think your alternative of having two functions that return each one attribute at a time is the best solution. Will look at it next week.